snippets/tree.js

```diff
@@ -4,6 +4,9 @@ var opts = {
   el: rootDiv,
   importURL: "./data/tree/B2014122194A560KL7I.4.ids.fa",
   bootstrapMenu: true,
+  conf: {
+    dropImport: true
```

Comment:
sorry, but as the msa is intended to be embedded we can't enable this by default